src/main/java/net/glowstone/entity/passive/GlowCow.java

```diff
@@ -19,6 +21,19 @@ public GlowCow(Location location) {
     }
 
     @Override
+    public void kill() {
+        super.kill();
+
+        Random r = new Random();
+
+        int beefCnt = r.nextInt(3);
+        int leatherCnt = r.nextInt(2);
+
+        getWorld().dropItemNaturally(getLocation(), new ItemStack(Material.RAW_BEEF, beefCnt));
```

Comment:
Your suggestion seems to me more complicated. Can you please provide further explanation why there should be a list?

Or do you recommend this

` public List<ItemsStack> getItemsForDropAfterDeath( deathreason )`